animal-options.lua

```diff
@@ -0,0 +1,80 @@
+--@ module = true
```

Comment:
There is related functionality in `zone.lua` (https://github.com/DFHack/dfhack/blob/develop/plugins/lua/zone.lua). I suggest you add this overlay there instead of creating a new top-level script.

Reply:
Should i just go ahead and close this pull request and make my changes there?

Reply:
Actually can I finish making changes here for review then after I will close?

Reply:
Sure